### PR TITLE
Implement `DriveUpdateState` service

### PR DESCRIPTION
**Pull request by jawid-h**

### CODE REVIEW

File: lib/services/driveUpdateState/DriveUpdateState.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+const { promisify } = require('util');
```
Comment (shumkov) [inline lib/services/driveUpdateState/DriveUpdateState.js]: Let's put sub-services into `drive` directory, as we did with DAPI

File: lib/services/driveUpdateState/DriveUpdateState.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        const response = await checkHealth(request);
+        if (response.getStatus() === healthCheckStatuses.SERVING) {
+          this.grpcClient = new UpdateStatePromiseClient(
+            this.getGrpcHostAndPort(),
+          );
+
+          return true;
+        }
+
+        await wait(1000);
```
Comment (shumkov) [inline lib/services/driveUpdateState/DriveUpdateState.js]: you can use `finally` if you are waiting anyways

**Comment by jawid-h** [comment]
@shumkov I guess we won't be able to test it until the new image of `Drive` is built.

**Comment by shumkov** [comment]
@jawid-h I've updated env variables please check https://github.com/dashevo/drive/pull/274

**Pull request merged**